azurecloudify/server.py

```diff
@@ -307,15 +359,46 @@ def _set_data_disk_json(vm_json, storage_account_name):
             storage_profile[constants.DATA_DISKS].append(curr_disk)
 
 
-def _get_vm_base_json(location, random_suffix_value, resource_group_name, storage_account_name, subscription_id,
-                      vm_name,availability_set_name):
-
+def _get_vm_base_json(location, random_suffix_value,
+                      resource_group_name, storage_account_name,
+                      subscription_id, vm_name):
```

Comment:
It was unused and there were calls to the function without it (a proper Python error).  Availability sets are obviously not supported, so why have partially implemented structures in the master branch?